core/src/main/java/org/apache/servicecomb/core/executor/GroupExecutor.java

```diff
@@ -106,6 +108,11 @@ public void initConfig() {
     LOGGER.info(
         "executor group={}. per group settings, coreThreads={}, maxThreads={}, maxIdleInSecond={}, maxQueueSize={}.",
         groupCount, coreThreads, maxThreads, maxIdleInSecond, maxQueueSize);
+    LOGGER.info("JDK standard thread pool rules:\n"
+        + "1.use core threads.\n"
+        + "2.if all core threads are busy, then queue the request.\n"
+        + "3.if queue is full, then create new thread util reach the limit of max threads.\n"
+        + "4.if queue is full, and threads count is max, then reject the request.");
```

Comment:
Do we change the stratege to create new thread first until the limit of max threads is reached. And then queue the requests?

Reply:
yes, i have wrote the reason in the PR comment
and i will copy it to jira task comment